app/src/main/java/org/wikipedia/talk/TalkTopicActivity.kt

```diff
@@ -466,14 +492,18 @@ class TalkTopicActivity : BaseActivity(), LinkPreviewDialog.Callback {
 
     companion object {
         private const val EXTRA_PAGE_TITLE = "pageTitle"
-        private const val EXTRA_TOPIC = "topicId"
+        const val EXTRA_TOPIC = "topicId"
+        const val EXTRA_SUBJECT = "subject"
+        const val EXTRA_BODY = "body"
         const val RESULT_EDIT_SUCCESS = 1
         const val RESULT_NEW_REVISION_ID = "newRevisionId"
 
-        fun newIntent(context: Context, pageTitle: PageTitle, topicId: Int, invokeSource: Constants.InvokeSource): Intent {
+        fun newIntent(context: Context, pageTitle: PageTitle, topicId: Int, invokeSource: Constants.InvokeSource, undoneSubject: String?, undoneBody: String?): Intent {
```

Comment:
You can set default ` = null` to the last two parameters so that you don't need to set `null` values in `TalkTopicsActivity.kt` and TalkTopicActivity.kt`